src/DesignQuery/scripts/cpp_script_for_sdt_generation/nlohmann_json/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
The library nlohmann_json is already included in FOEDAG. 
Do not add another copy here.
Please remove all the related files

Reply:
Extra copy of "nlohmann_json" library  has been removed